andrey_n/homework_5/homework_5/.gitignore

```diff
@@ -0,0 +1,8 @@
+.DS_Store
+.idea
+venv
+.coverage
+.coveragerc
```

Comment:
why did you add `. coveragerc` to gitignore?

Reply:
Hmm, It's custom file for me.

Reply:
why do you think so?

Reply:
Cuz I can include diff files if  I need exclude something from test_coverage

Reply:
why do you need to exclude something custom from your project directory?